test/counterParsing.cpp

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2019 Matthieu Gautier
```

Comment:
Wrong year

Reply:
Copy/Paste of the test file in libkiwix :)